### ISSUE TITLE
Export data from commnad line

### ISSUE THREAD

**Issue by andrewkip** [markummitchell/engauge-digitizer#262]
Hello! 
Thanks a lot for the great product  engauge-digitizer!
I have hundreds digitized files to export.
Is it possible to export data from commnad line in **_batch mode_**?
It would be great...
By the way ver. 10.7  under win10 64bit command line parameters strange behavior

engauge.exe "D:\TMP\2.dig"                           open 2.dig file
engauge.exe -open "D:\TMP\2.dig"                nothing happens
engauge.exe -open "D:\TMP\2.dig" -export "D:\TMP\2.csv"             nothing happens

**Comment by markummitchell** [comment]
You may be able to take advantage of some of the test functionality. You could create a script (in python for instance) that loop through each dig file. For each dig file, create an xml file (referred to as file.xml here) like:
```
<Script>
  <Cmd Type="CmdFileOpen" FileName="file1.dig" />
</Script>
```
Then run 'engauge -filecmdscript file.xml -regression -reset'. This command, and the xml file, will automatically export one file per curve. You may consider removing the '-reset' to get different behavior.

**Comment by markummitchell** [comment]
The strange behavior you mentioned is addressed in #263

**Comment by andrewkip** [comment]
Great! it's working.

It creates one csv file for all curves - it is ok.

And what about export settings like?
![default](https://user-images.githubusercontent.com/40334122/41562416-32883490-7355-11e8-8abc-b17c09401516.png)
How to set it via xml file?

**Comment by markummitchell** [comment]
I will see in a few hours

**Comment by andrewkip** [comment]
Thank you!

**Comment by markummitchell** [comment]
I can build a custom version for you immediately with the new capability for "Export Only" mode, or if you can wait a few days it will be included in a new release (along with 2 bugs I want to fix). Export Only will use the export settings. If you want the new mode now, please let me know what platform you are running on.

**Comment by markummitchell** [comment]
There is a WIndows build available at https://ci.appveyor.com/project/markummitchell/engauge-digitizer/branch/master/job/ogf2yb7r99uwo0rh/artifacts. Other builds will be available in an upcoming release.

**Comment by andrewkip** [comment]
Thank you! 
I've tested this functionality -exportonly - it works fine.

But. 
there is another problem:
i've file named 
SAM-145 06.18 Rev D-48 Avionics V521, V530, V550 Model RRJ-95B, RRJ-95LR-100, RRJ-95B-100 p 1 of 1  afm p 06.03 CTP 49 pdf 453.dig
yes it is long :)
after export engauge created file named 
SAM-145 06.csv  
it cuts the file name :(

-filecmdscript works fine with such long names 

And another problen If i have file name with cyrillic symbols engauge won't open it in comand line:
![default](https://user-images.githubusercontent.com/40334122/41766930-fb3827ac-7610-11e8-9b05-69ba1dd10f6e.png)

**Comment by markummitchell** [comment]
I have fixed the export file name issue as #267. I will work on the
cyrillic file name issue next.

Thanks for the good input.

**Comment by andrewkip** [comment]
Thank you for such quick reaction! 

But with this test build ( https://ci.appveyor.com/project/markummitchell/engauge-digitizer/branch/master/job/ogf2yb7r99uwo0rh/artifacts) I can't edit axis in my files.
It crashes silently, if I try to edit axis point (right mouse click) and it crashes with error report if I try to move axis point. 
I've attached error report and dig file.

[error_report.zip](https://github.com/markummitchell/engauge-digitizer/files/2130255/error_report.zip)
[SAM-145 06.3v Rev D-48 Model RRJ-95B p 1 of 1 FLAPS 3 afm p 06.03 CTP 9 pdf 373 Visota aerodroma.zip](https://github.com/markummitchell/engauge-digitizer/files/2130257/SAM-145.06.3v.Rev.D-48.Model.RRJ-95B.p.1.of.1.FLAPS.3.afm.p.06.03.CTP.9.pdf.373.Visota.aerodroma.zip)

**Comment by markummitchell** [comment]
I have determined that the file name with cyrillic characters is broken by
the Microsoft framework before it gets to any code that I have control
over. This means I cannot fix the problem. At the very start of the program
in Windows the file name appears as '??????.dig'. Further details are at
https://netbeans.org/nonav/bugzilla/show_bug.cgi?id=184513.

Some solutions are:
1) use only one type of character coding in the file name (since 'dig' is
required this would probably mean latin)
2) manually drag in the file after engauge has started
3) run in linux or osx where the problem does not appear

**Comment by andrewkip** [comment]
Ok I see. 
Thank you.

**Comment by markummitchell** [comment]
I spoke too soon. I may be able to open the file using the GetShortPathName
function to get a legal "8.3" format file name.

If this does not work, engauge should have a warning displayed so people
know what the problem is.

I will create an issue in github for this.

**Comment by markummitchell** [comment]
New issue #269 addresses your problem with the axis points. I will fix this
right away.

**Comment by markummitchell** [comment]
Something corrupted the point identiifers in your dig file. I tried to
recreate the problem using various locale settings and some other
strategies, but could not recreate the problem. It could be an issue with
your computer's configuration. I added a feature where the point
identifiers are checked after each operation, and if there is a problem
then a message immediately appears. This new feature is available at

https://ci.appveyor.com/project/markummitchell/engauge-digitizer/branch/master/job/emtp837tvmex835m/artifacts

It should be very helpful to identify the steps required to recreate the
problem. The information from the error_report.xml file you sent was not
enough to produce the problem - but that was helpful to eliminate from
consideration some failure modes I had been thinking about.
